### PR TITLE
add getUsers functionality - returning array of user objects

### PR DESCRIPTION
**Pull request by AlexMSm**
Tried testing for a 404 in the case of there being no users in the table, but I could not delete them as they are depended on by other tables. I'm not sure if there is a non-ridiculous way of testing this?

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -75,5 +75,26 @@ describe("app.js", () => {
       });
     });
   });
-  describe("/api/users", () => {});
+  describe("/api/users", () => {
+    describe("GET /api/users - returns array of user objects with propeties: ", () => {
```
Comment (CatHoang) [inline __tests__/app.test.js]: just a misspelling of properties

File: _controllers/app.controllers.js [hunk 2]
```diff
@@ -16,5 +16,11 @@ exports.getArticleById = (req, res, next) => {
 };
 
 exports.getUsers = (req, res) => {
```
Comment (CatHoang) [inline _controllers/app.controllers.js]: I would think about whether users should be in a different controller as it is we to be clear in our file structure

**Review by CatHoang — APPROVED** [review]
Hi Alex, nice clean code - just a couple of suggestions but feel free to merge once you have read over them.

Also I wouldn't worry about testing for no users :)

File: _controllers/app.controllers.js (hunk 2 above)
Comment (AlexMSm) [reply _controllers/app.controllers.js]: Noted - I thought this was being too verbose taking this approach at first but it makes sense.

**Pull request merged**